src/AiidaDFTK.jl

```diff
@@ -128,9 +128,14 @@ function run_json(filename::AbstractString; extra_output_files=String[])
     # Threading setup ... maybe later need to take parameters
     # from the JSON into account
     if mpi_nprocs() > 1
-        disable_threading()
-    else
-        setup_threading()
+        DFTK.setup_threading(; n_blas=1, n_fft=1)
```

Comment:
I think it makes sense to do this both with and without MPI, don't you think ?

Reply:
I can see the argument for it: AiiDA should already implement a first layer of scheduling and might not expect additional BLAS threads.